integreat_cms/cms/forms/translations/machine_translation_form.py

```diff
@@ -0,0 +1,112 @@
+import logging
+
+from django import forms
+from django.utils.translation import gettext_lazy as _
+
+from ...utils.translation_utils import mt_to_lang_is_permitted
+
+logger = logging.getLogger(__name__)
+
+
+class MachineTranslationForm(forms.Form):
+    """
+    Form for selecting target languages for machine translation of a content object.
+    """
+
+    automatic_translation = forms.BooleanField(
+        widget=forms.CheckboxInput(),
+        required=False,
+        label=_("Automatic translation"),
+        help_text=_(
+            "Tick if updating this content should automatically refresh or create its translations."
+        ),
+    )
+
+    translations_to_update = forms.ModelMultipleChoiceField(
+        widget=forms.CheckboxSelectMultiple(),
+        queryset=None,
+        required=False,
+        label=_("Update existing translations:"),
+    )
+
+    translations_to_create = forms.ModelMultipleChoiceField(
+        widget=forms.CheckboxSelectMultiple(),
+        queryset=None,
+        required=False,
+        label=_("Create new translations:"),
+    )
+
+    class Meta:
+        fields = [
+            "automatic_translation",
+            "translations_to_update",
+            "translations_to_create",
+        ]
+
+    def __init__(self, instance, region, language, **kwargs):
+        r"""
+        Initialize MT translation form
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+        """
+        super().__init__(**kwargs)
+
+        self.initial["automatic_translation"] = False
+
+        translation_targets = [
+            language
+            for language in region.language_tree_nodes.filter(parent=language.id)
+            if language.mt_provider and mt_to_lang_is_permitted(region, language.slug)
+        ]
```

Comment:
I was not aware, thanks to you and @JoeyStk further down for pointing this out.

And since using the prefetched tree isn't really less readable, even minimal performance improvement sounds good to me, so let's go with that version 😆